com.unity.visualeffectgraph/Editor/GraphView/Views/Properties/Vector2PropertyRM.cs

```diff
@@ -0,0 +1,126 @@
+using UnityEditor.UIElements;
+using UnityEditor.VFX.UI;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace UnityEditor.VFX.UI
+{
+    class Vector2PropertyRM : SimpleUIPropertyRM<Vector2, Vector2>
```

Comment:
Could you leave a comment telling what's the purpose of this custom PropertyRM?
(Unless this is against the coding practice at Unity ... I don't know actually)